### ISSUE TITLE
path not correctly built from URL

### ISSUE THREAD

**Issue by ronag** [Ethan-Arrowood/undici-fetch#22]
`url.pathname` does not directly correspond to `path`

```js 
      client.request({
        path: request.url.pathname,
        method: request.method,
        body: request.body,
        headers: request.headers,
        signal: init.signal
      }, (err, data) => {
        if (err) return reject(err)

        resolve(new Response(data.body, {
          status: data.statusCode,
          statusText: STATUS_CODES[data.statusCode],
          headers: data.headers
        }))
      })
```

Should be:

```js
...
path: `${url.pathname || ''}${url.search || ''}`,
...

```

**Comment by ronag** [comment]
Will probably be resolved through https://github.com/Ethan-Arrowood/undici-fetch/issues/14

**Comment by Ethan-Arrowood** [comment]
Thank you for pointing this out. Yeah once Agent is released on npm i'll be migrating over to that

**Comment by Ethan-Arrowood** [comment]
closed by #43
